paddle/fluid/framework/threadpool.cc

```diff
@@ -57,10 +57,10 @@ ThreadPool::ThreadPool(int num_threads) : running_(true) {
 ThreadPool::~ThreadPool() {
   {
     // notify all threads to stop running
-    std::lock_guard<std::mutex> l(mutex_);
+    std::unique_lock<std::mutex> l(mutex_);
```

Comment:
is unique_lock faster?

Reply:
No, here just want to be consistent with other place.